client.go

```diff
@@ -65,6 +66,8 @@ type Client struct {
 	sweeper     *sweep.Sweeper
 	executor    *executor
 
+	*liquidity.Manager
```

Comment:
I wouldn't expect this to be embedded in `Client`, but rather an object that exists side-by-side with `Client`? This is also what I meant in https://github.com/lightninglabs/loop/pull/265#discussion_r478287585

Reply:
Looks better outside of `Client` now!